src/routes/Bestenliste/+page.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+
+	import { highscore_table } from '$lib/stores/HighscoreStore.ts';
```

Comment:
An import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled.ts(5097)